src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -0,0 +1,243 @@
+        'registrationRequests': [device_3],
+        'grantRequests': [grant_request_3],
+        'conditionalRegistrationData': {}
+    }
+    cbsd_records_iter_2_domain_proxy_0 = {
+        'registrationRequests': [device_4],
+        'grantRequests': [grant_request_4],
+        'conditionalRegistrationData': {}
+    }
+    cbsd_records_iter_2_domain_proxy_1 = {
+        'registrationRequests': [device_5, device_6],
+        'grantRequests': [grant_request_5, grant_request_6],
+        'conditionalRegistrationData': {}
+    }
+    
+    # Protected entities records for multiple iterations
+    protected_entities_iter_1 = {
+        'palRecords': [pal_record_1],
+        'ppaRecords': [ppa_record_1], 
+        'escRecords': [esc_record_1]
+    }
+    protected_entities_iter_2 = {
+        'gwpzRecords': [gwpz_record_1], 
+        'fssRecords': [fss_record_1]
+    }
+
+    # SAS Test Harnesses configurations, 
+    # Following configurations are for two SAS test harnesses for two iterations
+    sas_test_harness_device_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    sas_test_harness_device_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    sas_test_harness_device_3 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    sas_test_harness_device_4 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_d.json')))
+    sas_test_harness_device_5 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    sas_test_harness_device_6 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_f.json')))
+
+    # Generate Cbsd FAD Records for SAS Test Harness 0, iteration 0
+    cbsd_fad_records_sas_test_harness_0_iteration_0 = generateCbsdRecords([sas_test_harness_device_1], [[grant_request_1]])
+
+    # Generate Cbsd FAD Records for SAS Test Harness 1, iteration 0
+    cbsd_fad_records_sas_test_harness_1_iteration_0 = generateCbsdRecords([sas_test_harness_device_4,sas_test_harness_device_5],
+                                                                          [[grant_request_2, grant_request_3], [grant_request_4]])
+
+    # Generate Cbsd FAD Records for SAS Test Harness 0, iteration 1
+    cbsd_fad_records_sas_test_harness_0_iteration_1 = generateCbsdRecords([sas_test_harness_device_2,sas_test_harness_device_3],
+                                                                          [[grant_request_1],[grant_request_2]])
+
+    # Generate Cbsd FAD Records for SAS Test Harness 1, iteration 1
+    cbsd_fad_records_sas_test_harness_1_iteration_1 = generateCbsdRecords([sas_test_harness_device_6], [[grant_request_5, grant_request_6]])
+   
+    # SAS Test Harnesses configuration  
+    sas_test_harness_0_config = {
+        'sasTestHarnessName': 'SAS-TH-1',
+        'hostName': 'localhost',
+        'port': 9001,
+        'serverCert': "certs/server.cert",
```

Comment:
Use os.path.join().

Reply:
Used   os.path.join for 'serverCert' , 'serverKey' and 'caCert'.